### PR TITLE
refactor: minting process, adding draft items

### PR DESCRIPTION
**Pull request by mmiranda-stackit**

### CODE REVIEW

File: src/common/minting/commands/common/activate-item-command.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        address: state.address,
+        image: { url: state.imageUrl, cid: state.cid },
+        tokenId: state.tokenId,
+      };
+
+      state.item = await this.itemService.activate(item);
+    } catch (ex) {
+      // TODO: Handle exceptions properly
+      console.log('ex|StoreNftCommand', ex);
+      return { ...state, error: MintError.StoreItemFailure };
```
Comment (AlefrankM) [inline src/common/minting/commands/common/activate-item-command.ts:33]: This is the incorrect error

File: src/common/minting/commands/common/store-ipfs-object-command.spec.ts [hunk 2]
```diff
@@ -38,31 +38,21 @@ describe('StoreIpfsObjectCommand', () => {
       it('should assign a RequiredBlobFile to state', async () => {
         const state = {} as MintState;
 
-        const expected = { error: MintError.RequiredBlobFile };
+        const expected = { ...state, error: MintError.RequiredBlobFile };
```
Comment (juliswer) [inline src/common/minting/commands/common/store-ipfs-object-command.spec.ts:41]: Why are you using copying an empty object inside another object?

**Review by juliswer — DISMISSED** [review]
Looks good. Great Job

**Review by AlefrankM — DISMISSED** [review]
LGTM >:(

File: src/common/minting/commands/common/store-ipfs-object-command.spec.ts (hunk 2 above)
Comment (mmiranda-stackit) [reply src/common/minting/commands/common/store-ipfs-object-command.spec.ts:41]: bcuz that's how the code being tested is behaving, not a particular reason tbh

**Pull request merged**